src/routes.js

```diff
@@ -181,22 +223,42 @@ const routes = [
     path: '/login',
     name: 'AccountLogin',
     component: AccountLogin,
-    meta: { requireLogin: false, requireAnonymity: true }
+    meta: {
+      requireLogin: false,
+      requireAnonymity: true,
+      title: { parts: () => [i18n.t('action.logIn')] }
```

Comment:
For posterity, we decided over Slack to use `action.logIn` rather than defining a separate `title.logIn` message. That's because "Log in" is the same whether or not it's in title case, and title case doesn't seem to be used in our other locales (and is maybe just uncommon in general?).